Ginger/GingerCoreNET/GeneralLib/EmailReadGmailOperations.cs

```diff
@@ -89,6 +103,48 @@ public Task ReadEmails(EmailReadFilters filters, EmailReadConfig config, Action<
             return Task.CompletedTask;
         }
 
+        private bool DoesSatisfyAttachmentFilter(MimeMessage message, EmailReadFilters filters)
+        {
+                 
+            if (string.IsNullOrEmpty(filters.AttachmentContentType))
+            {
+                return true;
+            }
+            IEnumerable<string> expectedContentTypes = null;
+            if (!string.IsNullOrEmpty(filters.AttachmentContentType))
+            {
+                expectedContentTypes = filters.AttachmentContentType.Split(";", StringSplitOptions.RemoveEmptyEntries);
+            }
+            if (expectedContentTypes != null && !expectedContentTypes.Any())
+            {
+                return true;
+            }
+
+            return HasAnyAttachmentWithExpectedContentType( message, expectedContentTypes);
+        }
+
+
+        private bool HasAnyAttachmentWithExpectedContentType( MimeMessage message,
+            IEnumerable<string> expectedContentTypes)
+        {
+            bool hasAnyAttachmentWithExpectedContentType = false;
+            IEnumerable<MimeEntity> attachments = message.Attachments;
+            int icount = attachments.Count();
+            if (icount > 0)
+            {
+                foreach (var attachment in attachments)
+                {
+                    String ContentType = ((MimePart)attachment).ContentType.ToString().Split(";", StringSplitOptions.RemoveEmptyEntries)[0].Split(":", StringSplitOptions.RemoveEmptyEntries)[1].Trim();
```

Comment:
Contentype is available in Contentype property of Attachment object. Please see 181 line.